cloud/azure/modules/app/variables.tf

```
@@ -123,3 +113,7 @@ variable "custom_hostname" {
   type        = string
   description = "custom hostname for the app to map the dns (used also for CORS)"
 }
+variable "key_vault_name" {
+  type = string
+  description = "Name of key vault where app secrets are stored."
```

suggestion (blocking): "app secrets" sounds a little similar to "app_key_secret_name" and "app-secret-key". Recommend avoiding "app" for referring to anything other than the CiviForm Java server. 
```suggestion
  description = "Name of key vault where secrets are stored."
```